src/test/java/ru/training/at/hw1/DataProviders.java

```diff
@@ -0,0 +1,13 @@
+package ru.training.at.hw1;
+
+public interface DataProviders {
+    String SUM_LONG_DATA_PROVIDER = "sum_long_data_provider";
+    String SUM_DOUBLE_DATA_PROVER = "sum_double_data_provider";
+    String SUB_LONG_DATA_PROVIDER = "sub_long_data_provider";
+    String SUB_DOUBLE_DATA_PROVIDER = "sub_double_data_provider";
+    String MUL_LONG_DATA_PROVIDER = "mul_long_data_provider";
+    String MUL_DOUBLE_DATA_PROVIDER = "mul_double_data_provider";
+    String DIV_LONG_DATA_PROVIDER = "div_long_data_provider";
+    String DIV_DOUBLE_DATA_PROVIDER = "div_double_data_provider";
+    String DIV_BY_ZERO_DATA_PROVIDER = "div_by_zero_data_provider";
```

Comment:
I see the names of the constants, but don't see appropriate keywords for a declaration.

Reply:
Fixed.